app/src/main/res/values/strings.xml

```diff
@@ -6,4 +6,10 @@
     <string name="speech_button">Speech</string>
     <string name="listening_text_indicator">Listening!</string>
     <string name="not_listening_text_indicator">not listening...</string>
+    <string name="what">Confused?</string>
+    <string name="info_text">Want to hear what some random Redditor thinks about something- just ask Snoo about your topic! Ask Snoo Anything returns Reddit comments related to what you say.</string>
+    <string name="more">Interested in trying out a feature we are working on? It optimizes by number of upvotes- but it is going to make things a bit slower.</string>
```

Comment:
(equivalent of <p>, then:)
Interested in trying out a feature we're working on? It optimizes responses by number of upvotes, but will make the app a bit slower.